cdm/core/src/test/java/thredds/filesystem/TestMFileOS.java

```diff
@@ -11,60 +11,81 @@
 import java.util.Random;
 import org.junit.ClassRule;
 import org.junit.Test;
+import org.junit.experimental.runners.Enclosed;
 import org.junit.rules.TemporaryFolder;
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
+import thredds.inventory.MFile;
 
-@RunWith(Parameterized.class)
```

Comment:
If you "hide whitespace changes" this diff is more readable-- I wrapped the parameterized tests in a inner class so I could run unparameterized tests as well